pl_bolts/utils/semi_supervised.py

```diff
@@ -24,14 +26,18 @@ class Identity(torch.nn.Module):
         model.fc = Identity()
 
     """
-    def __init__(self):
+    def __init__(self) -> None:
         super(Identity, self).__init__()
 
-    def forward(self, x):
+    def forward(self, x: Tensor) -> Tensor:
         return x
 
 
-def balance_classes(X: np.ndarray, Y: list, batch_size: int):
+def balance_classes(
+    X: Union[Tensor, np.ndarray],
+    Y: Union[Tensor, np.ndarray, Sequence[int]],
```

Comment:
I'm still concerned about if the annotated types include all the possibility, but I think these are fine for now.

Reply:
I will use this `balance_classes` and see if we can use other types as well.